[2022-07-22 13:00:28,675] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-03T00:00:00+00:00 [queued]>
[2022-07-22 13:00:28,778] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-03T00:00:00+00:00 [queued]>
[2022-07-22 13:00:28,779] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:28,779] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 13:00:28,780] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:28,851] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-07-03 00:00:00+00:00
[2022-07-22 13:00:28,881] {standard_task_runner.py:52} INFO - Started process 851 to run task
[2022-07-22 13:00:28,892] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-07-03T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmph706c8_p', '--error-file', '/tmp/tmp7be4312a']
[2022-07-22 13:00:28,903] {standard_task_runner.py:80} INFO - Job 73: Subtask t2
[2022-07-22 13:00:29,017] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 13:00:29,521] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-07-03T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 13:00:29,872] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-07-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-03T00:00:00+00:00
[2022-07-22 13:00:29,875] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 13:00:30,009] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220703T000000, start_date=20220722T130028, end_date=20220722T130030
[2022-07-22 13:00:30,244] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 13:00:30,394] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:18:10,465] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-03T00:00:00+00:00 [queued]>
[2022-07-25 09:18:10,533] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-03T00:00:00+00:00 [queued]>
[2022-07-25 09:18:10,535] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:10,535] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:18:10,536] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:10,601] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-07-03 00:00:00+00:00
[2022-07-25 09:18:10,607] {standard_task_runner.py:52} INFO - Started process 809 to run task
[2022-07-25 09:18:10,628] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-07-03T00:00:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmps6jngi4k', '--error-file', '/tmp/tmp8j281nn_']
[2022-07-25 09:18:10,632] {standard_task_runner.py:80} INFO - Job 70: Subtask t2
[2022-07-25 09:18:10,715] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:18:11,136] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-07-03T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:18:11,420] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-07-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-03T00:00:00+00:00
[2022-07-25 09:18:11,426] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:18:11,691] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220703T000000, start_date=20220725T091810, end_date=20220725T091811
[2022-07-25 09:18:11,741] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:18:11,906] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 12:26:56,598] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-03T00:00:00+00:00 [queued]>
[2022-07-25 12:26:56,690] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-03T00:00:00+00:00 [queued]>
[2022-07-25 12:26:56,692] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:26:56,694] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 12:26:56,694] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:26:56,827] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-07-03 00:00:00+00:00
[2022-07-25 12:26:56,844] {standard_task_runner.py:52} INFO - Started process 840 to run task
[2022-07-25 12:26:56,892] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-07-03T00:00:00+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpr11gjr76', '--error-file', '/tmp/tmpo8jf55jc']
[2022-07-25 12:26:56,900] {standard_task_runner.py:80} INFO - Job 114: Subtask t2
[2022-07-25 12:26:57,027] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 12:26:57,403] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-07-03T00:00:00+00:00 [running]> on host 929e6a3728d4
[2022-07-25 12:26:57,990] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-07-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-03T00:00:00+00:00
[2022-07-25 12:26:57,992] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 12:26:58,263] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220703T000000, start_date=20220725T122656, end_date=20220725T122658
[2022-07-25 12:26:58,430] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 12:26:58,599] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
